### ISSUE TITLE
Unable to resolve 'OpenIddict (>= 1.0.0-beta1)' for '.NETCoreApp,Versio n=v1.0

### ISSUE THREAD

**Issue by alvirtuoso** [piotrek-k/generator-dotnetcore-angular2-starterkit#6]
after running the yo generator, some dependencies in project.json  are doubled so I get duplicate object key error. example for doubled ones are:

   "Microsoft.AspNetCore.Server.IISIntegration": "1.0.0",
    "Microsoft.AspNetCore.Server.Kestrel": "1.0.1",
    "Microsoft.Extensions.Configuration.Json": "1.0.0",
    "Microsoft.Extensions.Logging": "1.0.0",
    "Microsoft.Extensions.Logging.Console": "1.0.0",
    "Microsoft.Extensions.Logging.Debug": "1.0.0",
    "Microsoft.Extensions.Options.ConfigurationExtensions": "1.0.0",
    "Microsoft.NETCore.App": { }

**Comment by piotrek-k** [comment]
Indeed, there are some duplicates, but in my case it doesn't cause any errors (just warnings)...
Did deleting those duplicates solve your problem?
